### DISCUSSION TITLE
🧐[问题]请问是路由配置侧边栏，是不支持单层吗？

### DISCUSSION THREAD

**Discussion by hengshanMWC** [umijs/umi#11155] [Q&A]
提问前先看看：

https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md

### 🧐 问题描述

当我配置只有一层的菜单栏，就会出现两个icon

依赖：
- @ant-design/pro-layout@7.5.0
- @umijs/max@4.0.41
我希望可以设置一层的菜单栏
### 🚑 其他信息

下面是配置了单层，会出现两个icon
<img width="242" alt="image" src="https://github.com/ant-design/pro-components/assets/45714422/bff49a80-e892-41ac-a6e2-f5ec979e7703">
<img width="268" alt="image" src="https://github.com/ant-design/pro-components/assets/45714422/88730b95-3dfa-4d79-a41a-85bed1647f76">
下面是两层，正常出行一个icon
<img width="275" alt="image" src="https://github.com/ant-design/pro-components/assets/45714422/11e20441-16a9-4313-b498-839768fb25d7">
<img width="268" alt="image" src="https://github.com/ant-design/pro-components/assets/45714422/1534dba8-4577-4ab8-a8c9-fa839c96446e">

**Comment by fz6m** [comment]
给一个最小复现看看。
